crates/holochain/src/core/workflow/validation_receipt_workflow.rs

```diff
@@ -77,15 +80,14 @@ pub async fn validation_receipt_workflow(
         if matches!(receipt.validation_status, ValidationStatus::Rejected) {
             // Block BEFORE we integrate the outcome because this is not atomic
             // and if something goes wrong we know the integration will retry.
-            conductor
-                .block(Block::new(
-                    BlockTarget::Cell(
-                        CellId::new((*dna_hash).clone(), author.clone()),
-                        CellBlockReason::InvalidOp(receipt.dht_op_hash.clone()),
-                    ),
-                    InclusiveTimestampInterval::try_new(Timestamp::MIN, Timestamp::MAX)?,
-                ))
-                .await?;
+            apply_block(Block::new(
+                BlockTarget::Cell(
+                    CellId::new((*dna_hash).clone(), author.clone()),
+                    CellBlockReason::InvalidOp(receipt.dht_op_hash.clone()),
+                ),
+                InclusiveTimestampInterval::try_new(Timestamp::MIN, Timestamp::MAX)?,
+            ))
+            .await?;
```

Comment:
Can't comment lower than this so the comment actually refers to 

```rust
return Ok(WorkComplete::Incomplete);
```

In the error case this will mean a problem in Lair can prevent validation receipts from proceeding with processing. For example, a missing public key in Lair would be enough to hit this branch and block all validation receipts from sending.